papas/data/eventplotter.py

```diff
@@ -0,0 +1,143 @@
+'''Papas event display'''
+from heppy.papas.data.historyhelper import HistoryHelper
+from heppy.display.core import Display
+from heppy.display.geometry import GDetector
+from heppy.display.trajectories import GHistoryBlock
+from ROOT import gPad
+
+class EventPlotter(object):
+
+    '''Papas event display.
+    '''
+    
+    def __init__(self, papasevent, detector, projections, directory):
+        '''Constructor.
+        
+        @param papasevent: event structure containing all the information from papas
+        @param detector: detector model used for the simulation
+        @param projections: a list of required projections, eg.
+        @param directory: output directory for images
+        '''
+        self.history = papasevent.history
+        self.papasevent = papasevent  
+        self.helper = HistoryHelper(papasevent)
+        self.detector = detector  
+        if projections is None:
+            projections = ['xy', 'yz']
+        self.projections = projections
+        self.initialized = False 
+        self.directory = directory
+            
+    def plot(self, plottype, screennames, particles_type_and_subtypes, 
```

Comment:
review interface (see my other comment). 

remove the to_file option and use the save method of display.core.Display
why the need for an EventPlotter class? should use Display (possibly modifying it). 
All display stuff should be in the display package.